gamestonk_terminal/main_helper.py

```diff
@@ -408,49 +408,85 @@ def load(other_args: List[str], s_ticker, s_start, s_interval, df_stock):
         return [s_ticker, s_start, s_interval, df_stock]
 
 
-def candle(s_ticker: str, s_start: str):
-    df_stock = trend.load_ticker(s_ticker, s_start)
-    df_stock = trend.find_trendline(df_stock, "OC_High", "high")
-    df_stock = trend.find_trendline(df_stock, "OC_Low", "low")
+def candle(s_ticker: str, other_args: List[str]):
+    """Shows candle plot of loaded ticker
+    Parameters
+    ----------
+    s_ticker: str
+        Ticker to display
+    other_args: str
+        Argparse arguments
+
+    """
+    parser = argparse.ArgumentParser(
+        prog="candle",
```

Comment:
prog goes right before description. So that we have add_help and formatter_class at start